XboxKeyboardMouse/Forms/Options.cs

```diff
@@ -717,12 +698,12 @@ public enum MouseButton {
         // -------------
 
         private void detachKeyCheckup(bool save = false) {
-            var k1 = (System.Windows.Input.Key)cfg.Controls_KB_Detach_MOD;
-            var k2 = (System.Windows.Input.Key)cfg.Controls_KB_Detach_KEY;
+            var k1 = (Key)cfg.Controls_KB_Detach_MOD;
+            var k2 = (Key)cfg.Controls_KB_Detach_KEY;
 
-            if (k2 == System.Windows.Input.Key.None & k1 != System.Windows.Input.Key.None) {
+            if (k2 == Key.None & k1 != Key.None) {
```

Comment:
You are doing a bitwise & instead of && AND

Reply:
@CallumCarmicheal

Reply:
@fqlx My mistake thanks :) 👍